CRM/Member/Form/MembershipView.php

```diff
@@ -280,7 +280,7 @@ public function preProcess() {
         $query .= " ORDER BY is_current_member DESC";
         $dao = CRM_Core_DAO::executeQuery($query);
         $related = [];
-        $relatedRemaining = CRM_Utils_Array::value('max_related', $values, PHP_INT_MAX);
+        $relatedRemaining = $values['max_related'] ?? PHP_INT_MAX;
```

Comment:
Note this is a functional change but without it you get a warning since $values['max_related'] can actually be null, if you haven't set a max on the membership type config, and then you _DO_ want it replaced by INT_MAX.